src/Microsoft.AspNetCore.Mvc.ApiExplorer/project.json

```diff
@@ -8,7 +8,9 @@
   "compilationOptions": {
     "warningsAsErrors": true,
     "keyFile": "../../tools/Key.snk",
-    "nowarn": [ "CS1591" ],
+    "nowarn": [
+      "CS1591"
+    ],
```

Comment:
Curious: Is the VS editor still doing the Wrong Thing:tm: with one-element arrays?